web/modules/custom/joinup_core/src/EventSubscriber/RefreshCachedFieldsEventSubscriber.php

```diff
@@ -86,89 +87,98 @@ public function __construct(EntityTypeManagerInterface $entityTypeManager, TimeI
   /**
    * {@inheritdoc}
    */
-  public function refreshExpiredField(RefreshExpiredFieldEventInterface $event) {
-    // Only react to fields that store the download count or visit count.
-    if (!in_array($event->getFieldName(), [static::FIELD_NAME_DOWNLOAD_COUNT, static::FIELD_NAME_VISIT_COUNT])) {
-      return;
+  public function refreshExpiredFields(RefreshExpiredFieldsEventInterface $event) {
+    $items = $event->getExpiredItems()->getItems();
```

Comment:
It could. I think this was based on the previous `::getExpiredItem()` method name. It is not an issue of Joinup though and should be fixed upstream.
*Reasons to rename:* Item being completely consistent with the return type.
*Reasons to not:* Try to imagine that the `Items` part of the `getExpiredItems` refers to `Items` which is a vague term for someone that is not familiar with the code and knows that it is a Collection Item. Thus, the developer needs to check what kind of return objects are returned. If it was `getExpiredEntities` then it would probably be in need of renaming. But `getExpiredItems` are referring to "Items" which has no direct explanation. So a glimpse into the return type is anyway needed.